stopping_lateral_distance: 1.35           # m
stopping_speed_limit: 1.0
current_pose_to_car_front: 4.0  # m
braking_reaction_time: 1.6
default_acceleration: 1.0
default_deceleration: 1.0


waypoint_loader:
  output_frame: map
  wp_left_width: 1.4
  wp_right_width: 1.4

lanelet2_global_planner:
  output_frame: map
  distance_to_goal_limit: 5.0

simple_local_planner:
  output_frame: map
  local_path_length: 100
  nearest_neighbor_search: kd_tree  # kd_tree. ball_tree, brute, auto
  transform_timeout: 0.06
  dense_waypoint_interval: 0.1
  tfl_maximum_deceleration: 2.8
  braking_safety_distance_obstacle: 4.0
  braking_safety_distance_stopline: 2.0
  braking_safety_distance_goal: 0.0